packages/paste-website/src/pages/components/form-pill-group/index.mdx

```diff
@@ -1,33 +1,53 @@
 ---
 title: Form Pill Group
 package: '@twilio-paste/form-pill-group'
-description: A Form Pill Group is an editable set of Pills used to visually represent a collection of entities inside a form field.
+description: A Form Pill Group is an editable set of Pills that represent a collection of selectable or removable objects.
```

Comment:
issue: same thing as in display pill group- preview of the site doesn't reflect this change.